.github/workflows/reproducible.yml

```diff
@@ -0,0 +1,46 @@
+name: "Reproducible build"
+on:
+  release:
+    types:
+      - created
```

Comment:
I have some doubts with this. According to https://docs.github.com/en/developers/webhooks-and-events/webhook-events-and-payloads#release, `created` means

> a draft is saved, or a release or pre-release is published without previously being saved as a draft

I think it isn't a common workflow for us, but will it trigger if we decide to publish a draft?

Reply:
Yes, already trigger it